Source/WebKit/UIProcess/ios/WKActionSheetAssistant.mm

```diff
@@ -42,6 +42,7 @@
 #import <WebCore/FloatRect.h>
 #import <WebCore/LocalizedStrings.h>
 #import <WebCore/PathUtilities.h>
+#import <pal/spi/ios/DataDetectorsUISoftLink.h>
```

Comment:
Nit: This header needs to go last, after all other (non-soft-link) headers (around line 59).

See other source files where `<pal/spi/ios/DataDetectorsUISoftLink.h>` is imported.